python-engine-test/build.gradle

```diff
@@ -15,6 +15,7 @@ test {
 }
 
 dependencies {
+  compile project(':Integrations')
```

Comment:
probably `testCompile` in keeping with the other deps in this project, since there is no non-test code

Reply:
(and technically we'd prefer testImplementation, but this project hasnt been updated to avoid that yet)